crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -37,14 +37,13 @@ pub struct DynamicEntity {
 
 impl DynamicScene {
     /// Create a new dynamic scene from a given scene.
-    pub fn from_scene(scene: &Scene, type_registry: &AppTypeRegistry) -> Self {
-        Self::from_world(&scene.world, type_registry)
+    pub fn from_scene(scene: &Scene) -> Self {
```

Comment:
Should this be a trait implementation? We could move this into an `impl From<&Scene> for DynamicScene` since it doesn't need the second argument anymore.

Reply:
Yeah this makes sense as well. I'll do this too!

Reply:
I also decided to hold off on this. If anything, we can save this for a future PR.

Reply:
Yeah, I'm fine to leave this to follow-up.